farmRe.py

```diff
@@ -0,0 +1,360 @@
+import statistics as stats
+import functions as functions
+
+class Farm:
```

Comment:
do that thing where it inherits the base class in python 
`Farm(something)` (I know it's the same behavior, default, but, just good to be verbose with that.

Reply:
farmRe is going to renamed farm.